config/manifests/bases/osp-director-operator.clusterserviceversion.yaml

```diff
@@ -6,48 +6,58 @@ metadata:
     capabilities: Seamless Upgrades
     operatorframework.io/suggested-namespace: openstack
     operators.operatorframework.io/builder: operator-sdk-v1.2.0
-    operators.operatorframework.io/internal-objects: '["overcloudipsets.osp-director.openstack.org","provisionservers.osp-director.openstack.org"]'
+    operators.operatorframework.io/internal-objects: '["openstackipsets.osp-director.openstack.org","openstackprovisionservers.osp-director.openstack.org"]'
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
   name: osp-director-operator.v0.0.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+    - description: VMSet represents a set of virtual machines hosts for a specific role within the Overcloud deployment
+      displayName: VMSet
+      kind: VMSet
+      name: openstackvmsets.osp-director.openstack.org
+      version: v1beta1
+    - description: OpenStackProvisionServer is the Schema for the provisionservers API
+      displayName: OpenStack Provision Server
+      kind: ProvisionServer
+      name: openstackprovisionservers.osp-director.openstack.org
+      version: v1beta1
     - description: OpenStackBaremetalSet represent a set of baremetal hosts for a specific role within the Overcloud deployment
-      displayName: OpenStackBaremetal Set
+      displayName: Open Stack Baremetal Set
```

Comment:
Is the spacing intended here and for `Open Stack Control Plane` below?

Reply:
This is auto generated. Let me see if I can fix it.